src/mscorlib/shared/System/IO/Path.Unix.cs

```diff
@@ -189,6 +189,19 @@ public static string GetPathRoot(string path)
             return IsPathRooted(path) ? PathInternal.DirectorySeparatorCharAsString : String.Empty;
         }
 
+        public static ReadOnlySpan<char> GetPathRoot(ReadOnlySpan<char> path)
+        {
+            ReadOnlySpan<char> result = new ReadOnlySpan<char>();
+
+            if (path == null)
```

Comment:
Testing Span against null is pointless. Span is never null because of it is a struct.

Reply:
(Multiple places)